lib/export/cimpl6Exporter.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * @file
+ * CIMPL 6 Exporter.
+ *
+ * This file provides the high level functions to export CIMPL models to
+ * CIMPL 6 format files. The primary usage of this export is to convert
+ * imported CIMCORE models into authorable CIMPL files.
+ *
+ */
+
+// const models = require('shr-models');
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+// const bunyan = require('bunyan');
+
+const { NamespaceFormatterCimpl6 } = require('./formatters/namespaceFormatterCimpl6');
+
+// var rootLogger = bunyan.createLogger({name: 'shr-text-import'});
+// var logger = rootLogger;
+
+// function setLogger(bunyanLogger) {
+//   rootLogger = logger = bunyanLogger;
+//   require('./formatters/namespaceFormatter').setLogger(logger);
+//   require('./formatters/mappingFormatter').setLogger(logger);
+//   require('./formatters/valueSetFormatter').setLogger(logger);
+//   require('./formatters/namespaceFormatter').setLogger(logger);
+// }
+
+/**
+ * The CIMPL 6 exporter
+ *
+ * CIMPL files are constructed and formatted
+ * by type ein individual formatter generators
+ *
+ * @param {Specifications} specs - The expanded specifications containing the cimpl models
+ *
+ */
+class Cimpl6Exporter {
+  constructor(specs) {
+    this._specs = specs;
+    this._namespaces = specs.namespaces.all;
+    this._elements = specs.dataElements.all;
+    this._valuesets = specs.valueSets.all;
+    this._mappings = specs.maps.targets.reduce((out, target) => {
+      out[target] = specs.maps.byTarget(target);
+      return out;
+    }, {});
+  }
+
+  get specs() { return this._specs; }
+  get namespaces() { return this._namespaces; }
+  get elements() { return this._elements; }
+  get valuesets() { return this._valuesets; }
+  get mappings() { return this._mappings; }
+
+  /**
+   * Primary export function. This triggers the chain of exporters and
+   * formatters to output DataElements, ValueSets, and Mappings by
+   * namespace.
+   *
+   * @param {string} path - the output destination for the formatted export.
+   */
+  exportToPath(path) {
+    const validNamespaces = this.namespaces.filter(ns => [''].indexOf(ns.namespace) == -1); //TODO: Verify this exclude primitives..?
```

Comment:
Yeah, so that code just checks if the namespace is '' and will filter it out. I changed it to be 
ns.namespace.trim().length > 0 but he was checking to see if the namespace was the empty string '' and then filtering it out when getting the validNamespaces to export to path.